src/ui/component/i18nMessage/view.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+import React from 'react';
+
+type Props = {
+  tokens: Object,
+  children: any,
```

Comment:
Could you add some examples of what `children` will look like in a comment next to this?

Reply:
I think this should be `children: string`